nginx.conf

```diff
@@ -26,18 +41,14 @@ server {
     location / {
         include uwsgi_params;
         uwsgi_pass unix:///tmp/kolibri_uwsgi.sock;
-        error_page 502 = @502;
+        error_page 400 401 402 403 404 405 406 407 408 409 410 411 412 413 414 415 416 417 418 421 422 423 424 426 428 429 431 451 500 501 502 503 504 505 506 507 508 510 511 @error500;
```

Comment:
I don't actually see handling for non-502 errors, with the generic design.

For now, maybe we should simply go back to handling 502 as a special case?

Reply:
Done at https://github.com/learningequality/kolibri-server/pull/35/commits/09e35ec33167f7eb1c7b7b62883d6dade0e344ca